packages/integration-sdk-runtime/src/logger/index.ts

```diff
@@ -399,20 +405,27 @@ export class IntegrationLogger
 
   validationFailure(err: Error) {
     const eventName = 'validation_failure';
-    const { errorId, description } = createErrorEventDescription(
+    const { errorId, errorCode, description } = createErrorEventDescription(
```

Comment:
All we get from the graph projects is an error during the validationFailure phase. We could enhance this but I don't know if it is necessary.